cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -185,6 +185,95 @@ <h2 class="header-slug">
         {% endfor %}
     {% endif %}
 
+
+    {# TODO: Remove the below when input-contains-icon
+             class is added cf-enhancements. #}
+    <div class="block">
+        <div class="block">
+            <div class="input-contains-icon">
+                <div class="input-contains-icon_before
+                            input-contains-icon_before__search">
+                    <span class="input-contains-icon_icon"></span>
```

Comment:
Do we really need the `span`s? Seems you could use the icon font mixins on the divs to create the icons in pseudo elems and reduce the markup.